corehq/apps/app_manager/models.py

```diff
@@ -6013,6 +6010,7 @@ def _set_versions(self, previous_version=None):
         self.set_media_versions(previous_version)
 
     @time_method()
+    @memoized
```

Comment:
this doesn't always work, I think because `previous_version` is a big scary doc, `@memoized` doesn't recognize duplicates.

Reply:
this comment looks weird in the big diff view - it was written on a commit which had `previous_version` as an arg to `create_all_files`, which I later removed